<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>****************************************************************************
**
** Copyright (C) 2013-2025 Donna Whisnant, a.k.a. Dewtronics.
** Contact: http://www.dewtronics.com/
**
** This file is part of the KJVCanOpener Application as originally written
** and developed for Bethel Church, Festus, MO.
**
** GNU General Public License Usage
** This file may be used under the terms of the GNU General Public License
** version 3.0 as published by the Free Software Foundation and appearing
** in the file gpl-3.0.txt included in the packaging of this file. Please
** review the following information to ensure the GNU General Public License
** version 3.0 requirements will be met:
** http://www.gnu.org/copyleft/gpl.html.
**
** Other Usage
** Alternatively, this file may be used in accordance with the terms and
** conditions contained in a signed written agreement between you and
** Dewtronics.
**
****************************************************************************</comment>
 <class>CDictionaryWidget</class>
 <widget class="QWidget" name="CDictionaryWidget">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="CDictionaryLineEdit" name="editDictionaryWord">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Dictionary Word to Search</string>
       </property>
       <property name="statusTip">
        <string>Enter word to search in the Dictionary</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonClearWord">
       <property name="toolTip">
        <string>Clear Word</string>
       </property>
       <property name="statusTip">
        <string>Clear Word</string>
       </property>
       <property name="icon">
        <iconset resource="KJVCanOpener.qrc">
         <normaloff>:/res/clear_left.png</normaloff>:/res/clear_left.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonHistoryBack">
       <property name="toolTip">
        <string>Word History Backword</string>
       </property>
       <property name="statusTip">
        <string>Go Backward in the Word History</string>
       </property>
       <property name="icon">
        <iconset resource="KJVCanOpener.qrc">
         <normaloff>:/res/Nav2_Arrow_Left.png</normaloff>:/res/Nav2_Arrow_Left.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonHistoryForward">
       <property name="toolTip">
        <string>Word History Forward</string>
       </property>
       <property name="statusTip">
        <string>Go Forward in the Word History</string>
       </property>
       <property name="icon">
        <iconset resource="KJVCanOpener.qrc">
         <normaloff>:/res/Nav2_Arrow_Right.png</normaloff>:/res/Nav2_Arrow_Right.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="definitionBrowser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Word Definition Browser</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDictionaryLineEdit</class>
   <extends>QTextEdit</extends>
   <header>DictionaryWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editDictionaryWord</tabstop>
  <tabstop>definitionBrowser</tabstop>
  <tabstop>buttonClearWord</tabstop>
  <tabstop>buttonHistoryBack</tabstop>
  <tabstop>buttonHistoryForward</tabstop>
 </tabstops>
 <resources>
  <include location="KJVCanOpener.qrc"/>
 </resources>
 <connections/>
</ui>
